content/docs/0.16.x/bridge/_index.md

```diff
@@ -0,0 +1,248 @@
+---
+title: Keptn Bridge (Web UI)
+description: The user interface of Keptn that presents all projects and services managed by Keptn. It is automatically installed with your Keptn deployment.
+weight: 8
+---
+
+## Views in Keptn Bridge
+
+The Keptn Bridge provides an easy way to browse all events that are sent within Keptn. When you access the Keptn Bridge, all projects will be shown on the start screen. When clicking on a project, the stages of this project, and all services are shown on the next view. An overview of those views is given below.
+
+### Project view
+
+* When you enter the Keptn Bridge and have not created a project yet, you will be guided to the instructions on how to setup a project and create a sample app.
+
+    {{< popup_image
+      link="./assets/project-dashboard-empty.png"
+      caption="Keptn Bridge empty project view">}}
+
+* If you have created projects, you will see the project listed including their stages.
+
+    {{< popup_image
+      link="./assets/project-dashboard.png"
+      caption="Keptn Bridge project view">}}
+
+### Environment view
+
+* When you select a project, you see the environment overview of the entire Keptn context.
+* This view allows to easily see the current deployed services on each stage and if there are any errors or pending approvals for one stage.
+  By clicking on a stage you can see more information about the deployed services on the specific stage on the right side.
+* The service details on the right are showing the current and the past 5 evaluations, open approvals and running remediations for each service.
+
+  {{< popup_image
+  link="./assets/environment-view.png"
+  caption="Keptn Bridge environment view">}}
+
+### Services view
+
+* One of the key steps for continuous delivery and remediation is the evaluation result based on SLO/SLIs. The services view gives you a clear overview
+  over the result of this evaluation step.
+* For each service the deployed artifacts are listed. Accordingly, for each artifact and executed stage, the evaluation result is displayed
+  as *HeatMap* and *Chart*:
+
+    {{< popup_image
+        link="./assets/services-view-heatmap.png"
+        caption="Evaluation Heatmap"
+        width="50%">}}
+
+    {{< popup_image
+        link="./assets/services-view-chart.png"
+        caption="Evaluation Chart"
+        width="50%">}}
+
+* To customize the chart, labels can be used. If the `sh.keptn.event.evaluation.triggered` event has the label `buildId` attached, the Keptn Bridge reads the value
+  of this label and uses it as label for the x-axis in the *Chart*. If the value of the label is an URL, the label will be displayed as a link, 
+  so you can easily link back to the Dynatrace Dashboard for example. 
```

Comment:
Do you want to call out Dynatrace here or say something more generic, like "link back to the dashboard of your observability server" ?